modules/serverless-invoking-python-functions.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-python-functions.adoc
+
+[id="serverless-invoking-python-functions_{context}"]
+= Invoking Python functions
```

Comment:
I believe there are other topics in the docs that aren't tasks and start with gerunds, e.g. [Understanding the update service](https://docs.openshift.com/container-platform/4.7/updating/understanding-the-update-service.html). I'm not really sure how else we would title this since it explains how a user would invoke functions?

Reply:
They shouldn't, though. I think those have slid in because "Understanding" makes it pretty clear that it's conceptual, where "Invoking" doesn't. (I might open a PR to s/Understanding/About someday.)

In the long run, if possible, it might be best to change this into a proper procedure. In the short term, I might call it something that makes it clearer that it's more conceptual than procedural, like:
- "Python function invocation"
- "About invoking Python functions"
- "Sample that invokes a Python function"